hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/Gateway.java

```diff
@@ -70,6 +75,17 @@ public Void call() throws Exception {
     OzoneConfigurationHolder.setConfiguration(ozoneConfiguration);
     UserGroupInformation.setConfiguration(ozoneConfiguration);
     loginS3GUser(ozoneConfiguration);
+
+    if (StringUtils.isEmpty(ozoneConfiguration.get(
```

Comment:
nit : can use `ServerUtils.getOzoneMetaDirPath(ozoneConfiguration)` instead.

Reply:
ServerUtils.getOzoneMetaDirPath throws an exception here. It would be better to set the value if not set.

Reply:
Got it thanks!